README.md

```diff
@@ -167,6 +167,19 @@ Breaking changes in shared libraries are addressed across all dependent projects
 The modular design allows for the easy addition of new projects or shared libraries without disrupting the existing structure.
 The use of Docker ensures that new projects can be deployed independently.
 
+## CI/CD architecture
+
+How could we use git diff to detect the changes in the monorepo and decide what the service to deploy are?
+
+Next step: create a script to run `git diff` command to identify the changed paths
```

Comment:
I think this is something we could probably look at in the future, but since we aren't generally automatically deploying anything yet, maybe it's not necessary.

It could potentially be useful to determine what needs to be built -- if nothing has changed, it doesn't need to be rebuilt -- on the other hand, if we're using layer caching and nothing has changed, a rebuild should be fast.

As far as running tests, I'm inclined to run all the tests using the code that's here, especially for the things that are kind of interdependent.